controllers/comments-controller.js

```diff
@@ -0,0 +1,17 @@
+const { fetchArticlesByID } = require("../models/article-models")
+const { fetchCommentsByArticleID } = require("../models/comment-models")
+
+function getCommentsByArticleID(request, response,next){
+    const {article_id} = request.params
+    const promises = [fetchCommentsByArticleID(article_id), fetchArticlesByID(article_id)]
+
+    Promise.all(promises)
```

Comment:
nicely done !

You could also chain the functions to verify if the article exists e.g.
fetchArticlesByID().then(() => { fetchComments etc.....

If fetchArticlesByID is successful the then block will run otherwise if it fails the catch block with will trigger with a error